src/openms/include/OpenMS/FORMAT/HANDLERS/MzMLHandler.h

```diff
@@ -5143,6 +5190,10 @@ namespace OpenMS
         if (j == 0)
         {
           os << "\t\t\t\t\t\t<cvParam cvRef=\"MS\" accession=\"MS:1000016\" name=\"scan start time\" value=\"" << spec.getRT() << "\" unitAccession=\"UO:0000010\" unitName=\"second\" unitCvRef=\"UO\" />\n";
+          if (spec.getDriftTime() > 0.0)
```

Comment:
yes, this is intended and has to do with the fact that I found some cases where the drift time was *not* stored in the precursor attribute but directly in the spectrum itself. the best solution to keep such mzML files more or less consistent was to store the drift time as a property of the spectrum but allow different precursors to have different drift times (e.g. in principle it is possible to have a spectrum that is a mixture of multiple precursors that all have different drift times - in this case, these would all get stored in each precursor and the spectrum drift time would be somewhat meaningless as it is simply the last read precursor).